### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by borjanpopovic**
Added a search bar for location list.

Map part 2: Added  address search to location form. Location-position  for the map can be determined  by typing in a (full) address or city name. After entering a click somewhere outside the input field for the address or enter key press should be performed to apply the coordinates of the entered address.  

Thanks for reviewing!

### CODE REVIEW

File: src/Components/FooterNavigation.js [hunk 1]
```diff
@@ -45,13 +45,14 @@ const NavBox = styled.div`
...
     background: var(--secondary);
-    opacity: 0.9;
+    opacity: 0.8;
   }
 `;
 
 const NavLinkStyled = styled(NavLink)`
   background: transparent;
-  box-shadow: 0 0.125rem 0.75rem hsla(213, 52%, 20%, 0.2);
+  //box-shadow: 0 0.125rem 0.75rem hsla(213, 52%, 20%, 0.2);
```
Comment (heleah) [inline src/Components/FooterNavigation.js]: you can remove this comment (:

File: src/Components/LocationForm.js [hunk 2]
```diff
@@ -18,32 +16,34 @@ export default function LocationForm({ onAddLocations }) {
   const [isError, setIsError] = useState(false);
   const [status, setStatus] = useState(null);
 
-  /* console.log('added location', location); */
+  console.log('added location', location);
```
Comment (heleah) [inline src/Components/LocationForm.js]: please consider removing this console.log

File: src/Components/FooterNavigation.js [hunk 3]
```diff
@@ -45,13 +45,14 @@ const NavBox = styled.div`
...
+  box-shadow: 0 0.11rem 0.1rem var(--secondary-dark);
```
Comment (heleah) [inline src/Components/FooterNavigation.js]: "box" comes after "bor..." in alphabet (:

File: src/Components/LocationForm.js [hunk 4]
```diff
@@ -149,18 +151,18 @@ const Form = styled.form`
   box-shadow: 0 0.125rem 0.75rem hsla(213, 52%, 20%, 0.2);
   color: var(--border-dark);
   font-size: 1rem;
-  place-items: center;
   gap: 0.5rem;
   padding: 1rem;
-  margin-top: 3rem;
+  place-items: center;
   margin: 1rem;
+  margin-top: 3rem;
```
Comment (heleah) [inline src/Components/LocationForm.js:154]: m before p (:

File: src/Components/FooterNavigation.js (hunk 1 above)
Comment (borjanpopovic) [reply src/Components/FooterNavigation.js]: Thank you for the hint! Consider it done. :)

File: src/Components/LocationForm.js (hunk 2 above)
Comment (borjanpopovic) [reply src/Components/LocationForm.js]: Thank you. I'm gonna delete it.

File: src/pages/map.js [hunk 5]
```diff
@@ -53,23 +53,30 @@ const H1 = styled.h1`
...
     bottom: 0;
-    height: 88vh;
+    height: 85vh;
     margin-top: 1rem;
     width: 100%;
     z-index: 0;
   }
 
+  .leaflet-popup-content-wrapper {
+    /* background-color: transparent; */
```
Comment (heleah) [inline src/pages/map.js]: remove comment

File: src/Components/FooterNavigation.js (hunk 3 above)
Comment (borjanpopovic) [reply src/Components/FooterNavigation.js]: Whoops. Overlooked this one. Thanks :)

File: src/Components/LocationForm.js (hunk 4 above)
Comment (borjanpopovic) [reply src/Components/LocationForm.js:154]: This one as well :)

**Review by heleah — APPROVED** [review]
there's a few comments to be removed and css properties to order alphabetically, but otherwise lgtm (: ✨

File: src/pages/map.js (hunk 5 above)
Comment (borjanpopovic) [reply src/pages/map.js]: Thx! Gonna remove it.

**Comment by borjanpopovic** [comment]
Many thanks for your review!

**Pull request merged**